apps/i18n/javalab/en_us.json

```diff
@@ -3,6 +3,7 @@
   "allFilesCompile": "All Files Compile!",
   "backpackLabel": "Backpack",
   "clearConsole": "Clear Console",
+  "commentSaveError": "There has been an error resolving or deleting this comment, please try again.",
```

Comment:
should we have different strings for resolve and delete? It may be confusing to see 'delete' when you don't have the option to delete a comment

Reply:
Probably, I was just trying to put a bare minimum error message up. Will look at updating